dns/bind/src/opnsense/mvc/app/models/OPNsense/Bind/General.xml

```diff
@@ -19,6 +19,10 @@
             <Required>Y</Required>
             <asList>Y</asList>
         </listenv6>
+        <port type="PortField">
+            <default>53530</default>
```

Comment:
looks a bit like mdns;)

Reply:
@fichtner really bump on a dev version? @fabianfrz 5353 is mdns? I sent it to a non standard so it wont interfere with unbound/dnsmasq

Reply:
yes or maybe. it's up to you. bump is safer.